locales/de.json

```diff
@@ -1462,7 +1462,7 @@
 		"label": {
 			"newPassword": "Neues Passwort: ",
 			"repeatYourNewPassword": "Wiederhole dein neues Passwort:",
-			"under14YearsOld": "unter 14 Jahre alt."
+			"under14YearsOld": "13 Jahre oder jünger."
```

Comment:
please change key to under13YearsOld

Reply:
key should stay like this as under 14 means 13 or younger